Assets/Scripts/Augment/BulletModifiers/DecalModifier.cs

```diff
@@ -49,6 +49,10 @@ public void Detach(ProjectileController projectile)
 
     private void OnHit(Collider target, ref ProjectileState state)
     {
+        // Soda cans are tiny and flail around, making them unsuitable for placing bullet holes on
+        if (target.TryGetComponent<SodaCan>(out var _))
```

Comment:
Hmmm, maybe we should have some kind of way to mark certain objects as "not allowed as decal parents" in the furture? I don't think it's necessary yet ofc

Reply:
Yeah, and I think it'd also be useful to make certain objects have decals that display *only on that object*, for things that move around. It was absolutely necessary to add this check for the soda cans since otherwise they'd roll around and project bullet holes onto everything in a distractingly unrealistic manner.